x-pack/metricbeat/module/aws/cloudwatch/cloudwatch.go

```diff
@@ -288,6 +290,12 @@ func (m *MetricSet) readCloudwatchConfig() (listMetricWithDetail, map[string][]n
 	resourceTypesWithTags := map[string][]aws.Tag{}
 
 	for _, config := range m.CloudwatchConfigs {
+		// If tags_filter on metricset level is given, overwrite tags in
```

Comment:
Can we somehow document this behaviour in the docs?:)

Reply:
Yeah I'm actually gonna deprecate `tags` config in PR https://github.com/elastic/beats/pull/16733. That's why I didn't add any doc around this 😄

Reply:
👍